pyproject.toml

```diff
@@ -24,6 +24,8 @@ accept-types = "^0.4.1"
 isamples_metadata = {git = "https://github.com/isamplesorg/metadata.git", branch = "develop"}
 shapely = "^1.7.1"
 geojson = "^2.5.0"
+Jinja2 = "^3.0.1"
+heartrate = "^0.2.1"
```

Comment:
Had to add these two lines since poetry has a bug where it misses transitive dependencies.  Explicitly adding them to the list makes the `requirements.txt` export correctly.